app/src/components/Forms/ContributionReady/index.js

```diff
@@ -1,342 +1,105 @@
-    display: grid;
-    grid-template-rows: repeat(auto-fit(50px, 1fr));
-    grid-template-columns: 1fr;
-    grid-gap: 20px;
-    margin-bottom: 20px;
-  `,
-  cityStateZip: css`
-    width: 96%%;
-    min-height: 25px;
-    display: grid;
-    grid-template-rows: repeat(auto-fit, minmax(15px, 1fr));
-    grid-template-columns: 2fr 22% 24%;
-    grid-gap: 20px;
-    margin-bottom: 20px;
-  `
-};
+const onSubmit = (data) => {
+  console.log(data)
+}
 
-const headerStyles = {
-  header: css`
-    display: flex;
-    justify-content: space-between;
-    margin-right: 38px;
-    font-family: Rubik;
-    font-style: normal;
-    font-weight: normal;
-  `,
-  leftColumn: css`
-    margin-right: 70px;
-  `,
-  rightColumn: css`
-    display: flex;
-    flex-direction: row-reverse;
-    align-items: flex-end;
-  `,
-  invoice: css`
-    font-size: 48px;
-    line-height: 57px;
-    margin: 0px;
-    /* identical to box height */
-    color: #333333;
-  `,
-  subheading: css`
-    font-size: 16px;
-    line-height: 19px;
-    margin-top: 0px;
-    width: 360px;
-  `,
-  labelBlock: css`
-    margin-right: 40px;
-  `,
-  labels: css`
-    font-size: 13px;
-    line-height: 15px;
-    color: #979797;
-    margin-bottom: 4px;
-  `,
-  smallBlueText: css`
-    font-size: 13px;
-    line-height: 15px;
-    margin: 0px;
-    /* Link */
-    color: #5f5fff;
-  `,
-  largerBlueText: css`
-    font-size: 16px;
-    line-height: 19px;
-    margin: 0px;
-    /* Link */
-    color: #5f5fff;
-  `,
-  status: css`
-    font-size: 13px;
-    line-height: 15px;
-    color: #979797;
-    margin-bottom: 4px;
-  `,
-  actualStatus: css`
-    font-size: 21px;
-    line-height: 25px;
-    color: #000000;
-    margin-top: 0px;
-    margin-bottom: 20px;
-  `,
-  statusBlock: css`
-    flex-direction: column;
-    align-items: left;
-  `,
-  buttonDiv: css`
-    display: flex;
-    justify-content: space-between;
-    align-self: flex-end;
-  `,
-  submitButton: css`
-    background-color: #d8d8d8;
-    border-radius: 5px;
-    color: white;
-    width: 165px;
-    height: 50px;
-  `,
-  draftButton: css`
-    // HAVING ZERO IMPACT
-    background-color: #5f5fff;
-    border-radius: 5px;
-    width: 165px;
-    height: 50px;
-  `
-};
+// TODO: need to get data for the following:
+// - oaeContributionType
+// - paymentMethod
+// - employerZipcode
+// - occupationLetterDate
+// - linkToDocumentation
+// - notes
+const mapDataToForm = (contribution) => {
+  console.log(contribution)
+  const {
+    date,
+    createdAt,
+    type, 
+    subtype,
+    contributorType,
+    amount, 
+    checkNumber,
+    firstName, 
+    lastName, 
+    address1,
+    address2, 
+    city, 
+    state,
+    zip,
+    email,
+    phone,
+    phoneType, 
+    occupation,
+    employerName,
+    employerCity,
+    employerState,
+    calendarYearAggregate,
+    inKindDescription,
+  } = contribution
+  return {
+    // BASICS VALUES
+    dateOfContribution: format(new Date(date), "YYYY-MM-DD"),
```

Comment:
Complains about YYYY-MM-DD should  be yyyy-MM-dd

Reply:
I haven't been able to replicate this one... Is the complaint coming from the console?